db-setup-testdata.sql

```diff
@@ -0,0 +1,44 @@
+INSERT INTO scripts VALUES (1, 'test');
```

Comment:
I found myself using this over and over to reset the database state, especially after changing the schema so that I would have quick data available. If we find this is no longer needed we can always delete it or maybe even use it into for integration tests.

Reply:
That makes sense, sounds good to me!